robot/Cumulus/resources/NPSP.py

```diff
@@ -498,7 +498,14 @@ def verify_related_list_field_values(self, **kwargs):
             locator= npsp_lex_locators['record']['related']['field_value'].format(name,value)
             self.selenium.wait_until_page_contains_element(locator)
             self.selenium.page_should_contain_element(locator)
-            
+     
+    def verify_related_object_field_values(self, **kwargs):
+        """verifies the specified field,value pairs in the related object page (table format)""" 
+        for name, value in kwargs.items():
+            locator= npsp_lex_locators['object']['field-value'].format(name,value)
+            self.selenium.wait_until_page_contains_element(locator)
+            self.selenium.page_should_contain_element(locator)
```

Comment:
You don't need both commands. Either of them is fine. Both the commands, give the ability to print a custom message/error to the console in case it fails (Something similar to what Bryan was mentioning during our last Robot check-in).
- wait_until_page_contains_element(locator, error="<custom message>")
- page_should_contain_element(locator, message = "<custom message>")

Reply:
thanks for catching this, I copied an existing keyword I had created long back and didn't pay attention. Will update it.